lib/web_ui/lib/src/engine/semantics/scrollable.dart

```diff
@@ -28,11 +28,31 @@ import 'semantics.dart';
 /// viewport "scrollTop" may take positive values.
 class Scrollable extends RoleManager {
   Scrollable(SemanticsObject semanticsObject)
-      : super(Role.scrollable, semanticsObject);
+      : super(Role.scrollable, semanticsObject) {
+        _scrollOverflowElement.style
+        ..position = 'absolute'
+        ..transformOrigin = '0 0 0'
+        // Ignore pointer events since this is a dummy element.
+        ..pointerEvents = 'none';
+        semanticsObject.element.append(_scrollOverflowElement);
+      }
 
   /// Disables browser-driven scrolling in the presence of pointer events.
   GestureModeCallback? _gestureModeListener;
 
+  /// Dummy element used as a workaround for issue
```

Comment:
Unfinished sentence. Also nit, this element is not exactly a dummy, since it plays an important role.